.github/workflows/ci.yaml

```diff
@@ -36,6 +36,23 @@ jobs:
       target-branch: ${{ inputs.target-branch }}
     secrets: inherit
 
+  build-kata-static-tarball-s390x:
+    uses: ./.github/workflows/build-kata-static-tarball-s390x.yaml
+    with:
+      tarball-suffix: -${{ inputs.tag }}
+      commit-hash: ${{ inputs.commit-hash }}
```

Comment:
Should we also pass through `target-branch: ${{ inputs.target-branch }}` which is needed for the rebase?